filebeat/docs/inputs/input-journald.asciidoc

```diff
@@ -128,11 +128,27 @@ multiple log messages are written to a journal while {beatname_uc} is down,
 only the last log message is sent on restart.
 * `cursor`: On first read, starts reading at the beginning of the journal. After
 a reload or restart, continues reading at the last known position.
+* `since`: Use the `since` option to determine where to start reading from.
 
 If you have old log files and want to skip lines, start {beatname_uc} with
 `seek: tail` specified. Then stop {beatname_uc}, set `seek: cursor`, and restart
 {beatname_uc}.
 
+[float]
+[id="{beatname_lc}-input-{type}-cursor_seek_fallback"]
+==== `cursor_seek_fallback`
+
+The position to start reading the journal from if no cursor information is
+available. Valid options are `head`, `tail` and `since`.
+
+[float]
+[id="{beatname_lc}-input-{type}-since"]
+==== `since`
+
+A time offset from the current time's record to start reading from. To use
```

Comment:
I was struggling with that sentence. Maybe I am missing some subtle context around "record". How this?

```suggestion
A time offset from the current time to start reading from. To use
```

Reply:
I was struggling with this too. Yours is good; it's not quite what is meant, but it will do.